src/UI/ContentPlaceholders.elm

```diff
@@ -0,0 +1,236 @@
+module UI.ContentPlaceholders exposing
```

Comment:
I think the name should be in singular form like the rest of the components. The showcase module is fine though